app/main.py

```diff
@@ -1,3 +1,5 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    small_phrase = phrase.lower()
```

Comment:
The conversion to lowercase is correctly done for `phrase`. Ensure that `phrase` is a string to avoid any runtime errors.